README.md

```diff
@@ -65,7 +65,15 @@ for await (const data of body) {
 console.log('trailers', trailers)
 ```
 
-Using [the body mixin from the Fetch Standard](https://fetch.spec.whatwg.org/#body-mixin).
+## Body Mixins
+
+The `body` mixins are the most common way to format the request/response body from a [`ReadableStream`](https://streams.spec.whatwg.org/#readablestream). Mixins include:
```

Comment:
I don't think it's correct. the request method does not have anything to do with WHATWG `ReadableStream`.

Reply:
totally open to suggestion. was a best-stab at it without intimate knowledge, but to show how we could be thorough

Reply:
I would just remove the reference to ReadableStream.

Reply:
```suggestion
The `body` mixins are the most common way to format the request/response body. Mixins include:
```

Reply:
@ronag please reopen this so I can apply the suggestion above